addons/sourcemod/scripting/nd_discord_playerlist.sp

```diff
@@ -0,0 +1,192 @@
+Handle SaveTeamsTimer;
+bool SavingTeams;
+Handle DatabaseHandle;
+
+public Plugin myinfo =
+{
+    name        = "[ND] Discord Player List",
+    author      = "zookatron",
+    description = "Provides player list information to Discord",
+    version     = "dummy",
+    url         = "https://github.com/stickz/Redstone/"
+};
+
+public void OnPluginStart()
+{
+    if (!SQL_CheckConfig("discord"))
+    {
+        SetFailState("Could not find configuration for \"discord\" database");
+        return;
+    }
+
+    SQL_TConnect(OnDatabaseConnected, "discord");
+
+    HookEvent("player_team", Event_PlayerTeam);
+}
+
+public Action Event_PlayerTeam(Event event, const char[] name, bool dontBroadcast)
+{
+    SaveTeams();
+    return Plugin_Continue;
+}
```

Comment:
This should be replaced with `#include <nd_team_eng>` then `public void ND_OnPlayerTeamChanged(int client, bool valid)`. This DOES cover bots. valid will be false if the player is a bot.

We can still keep `OnClientPutInServer(int client)` and `OnClientDisconnect_Post(int client)`.

Reply:
done